diagnostic.md

```diff
@@ -36,21 +36,21 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+relative path
```

Comment:
This is in fact an absolute path due to it starting at the root. You tell this by `/Users/blah_blah/`. Anytime you start at the root, it is an absolute path.